assets/css/components/_news-and-views.scss

```diff
@@ -31,6 +32,7 @@
     grid-template-columns: repeat(2, 1fr);
 
     figure {
+      display: block !important;
```

Comment:
Why did you use `!important` here?

Reply:
Was only able to get this to work by using !important. I know it is not recommended to use this so lets discuss work arounds during todays We Count check in.